src/main/kotlin/racing/controller/RacingCarController.kt

```diff
@@ -0,0 +1,22 @@
+package racing.controller
+
+import racing.service.RacingCarService
+import racing.view.InputView
+import racing.view.ResultView
+
+class RacingCarController {
+    private val racingCarService = RacingCarService()
```

Comment:
주생성자를 통해서 프로퍼티를 주입받을 수 있도록하면 (지금은 테스트하고있지 않지만)  테스트하기가 더 쉬울 것 같아요. 🤗

```suggestion
class RacingCarController(private val racingCarService = RacingCarService()) {
```